scripts/pycharm-launch.py

```diff
@@ -73,11 +73,17 @@ def write_required_statement(out=sys.stderr):
 sys.path.insert(0, abspath)
 if not parsed.no_config:
     if not os.environ.get('AGENT_CONFIG'):
-        if not os.path.exists(os.path.join(abspath, 'config')):
+        path_found = None
+        # Order of search is as follows config, config.yml, config.json
```

Comment:
It would be worth mentioning this in readthedocs

Reply:
yup I'll add it